README.md

```diff
@@ -9,39 +9,30 @@ To install this cnab bundle, you can use any available cnab runtime. The followi
 ### Steps
 1. Download the latest duffle release for your operating system from duffle's [release page](https://github.com/deislabs/duffle/releases),
  put it somewhere on your path, and make it executable using, for example, `chmod +x /usr/local/bin/duffle`.
-1. Start a kubernetes cluster as described in the riff [getting started](https://projectriff.io/docs/getting-started/) documentation,
-but stop once you've created a cluster and, if appropriate, given yourself cluster-admin permissions.
- If using `minikube`, please run the following command before starting `minikube`:
-    ```
-    minikube config set embed-certs true
-    ```
-1. Check that your kubeconfig file is configured to talk to that cluster:
-    ```
-    kubectl config current-context
-    ```
+1. Start a ***local*** kubernetes cluster (minikube OR docker-for-desktop) by following the instructions in [getting started](https://projectriff.io/docs/getting-started/minikube/).
```

Comment:
let's document the normal case and the caveat. The restriction is not whether the cluster is local, it's how authenticating to the cluster is managed.

Reply:
re-did the readme per cluster type.